hugo/content/contact.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contact
+toc: true
```

Comment:
This seems to enable a table of content.

But I do not see any toc being rendered!?

Reply:
Well I haven't checked this in the docs, but with other software I use (for some other projects) this tag can have different meanings, for example it means to enable an indexing of the website (so that you have section links to each category on the page).

Reply:
It’s a parameter that we currently do not use. If we were to enable it here, it could lead to unforeseen consequences once we add toc handling to the layout templates. We don’t know what would happen.

It should be removed to prevent unknown risks while not adding or doing anything.

Reply:
Ok, just for clarification:
1. Without this option we would still have section links like: https://www.mumble.info/contribute/#report-bugs-and-feature-requests ?
2. Is this a custom parameter?
3. This parameter is used on other pages too, should we remove it everywhere?

Reply:
Yes, [contribute.md](https://github.com/mumble-voip/mumble-www/blob/master/hugo/content/contribute.md) manually creates a list with links as content, and depends on the headline text which leads to an id attribute to match the link that is declared.

If toc were implemented like this, we would suddenly have two table of content lists.

If a headline changes, the manual list has to be updated too atm.

When I checked our layout did not have toc handling. So yes, it should IMO be removed from other pages too.

Given that the params docs gives an example for `include_toc` as a parameter, I think `toc` as it was used here is a user defined parameter too. There seems to be no other doc on `toc`, which supports that theory.